spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,20 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      post :lock_account, id: other_user.id
+      expect(other_user.reload.access_locked?).to be_truthy
+    end
+  end
+
+  describe '#unlock_account' do
+    it 'it unlocks the given account' do
+      other_user = FactoryGirl.create :user
+      post :unlock_account, id: other_user.id
+      expect(not(other_user.reload.access_locked?)).to be_truthy
```

Comment:
As said before use `!`, not `not` for negation. But here again there's a better way than negation in the first place and that is to use the `be_falsey` matcher.

Besides this spec is not testing much. You start with a user account that's not lock, run some code and then check whether the user account is not locked. So while you ensure that the code you run does not lock the account, you do not ensure that it unlocks the account.